scripts/check-env/check-cs-env.js

```diff
@@ -40,6 +43,57 @@ async function checkEnv() {
     }
 }
 
+async function checkDiskSpaceFullEnv() {
+    this.alfrescoJsApi = new alfrescoApi.AlfrescoApiCompatibility({
+        provider: 'ECM',
+        hostEcm: program.host
+    });
+
+    await this.alfrescoJsApi.login(program.username, program.password);
+
+    let folder;
+
+    try {
+        folder = await alfrescoJsApi.nodes.addNode('-my-', {
+            'name': `try-env`,
+            'relativePath': `Builds`,
+            'nodeType': 'cm:folder'
+        }, {}, {
+            'overwrite': true
+        });
+
+    } catch (error) {
+        folder = await alfrescoJsApi.nodes.getNode('-my-', {
+            'relativePath': `Builds/try-env`,
+            'nodeType': 'cm:folder'
+        }, {}, {
+            'overwrite': true
+        });
+    }
+
+    try {
+        let pathFile = path.join(__dirname, '../../', 'README.md');
+        let file = fs.createReadStream(pathFile);
+
+        let uploadedFile = await alfrescoJsApi.upload.uploadFile(
+            file,
+            '',
+            folder.entry.id,
+            null,
+            {
+                'name': 'README.md',
+                'nodeType': 'cm:content',
+                'autoRename': true
+            }
+        );
+
+        this.alfrescoJsApi.node.deleteNode(uploadedFile.entry.id, {permanent: true});
+    } catch (error) {
+        console.log('Is not possible upload a new file possible content full');
```

Comment:
can we have an error message that will be easy to spot in a log file?
e.g. 
================= ================= ==================
================ Not able to upload a file ===================
============== Make sure Content is not full =================
================= ================= ==================

Reply:
Why not a dragon?
```javascript  
(  )   /\   _                 (     
    \ |  (  \ ( \.(               )                      _____
  \  \ \  `  `   ) \             (  ___                 / _   \
 (_`    \+   . x  ( .\            \/   \____-----------/ (o)   \_
- .-               \+  ;          (  O                           \____
                          )        \_____________  `              \  /
(__                +- .( -'.- <. - _  VVVVVVV VV V\                 \/
(_____            ._._: <_ - <- _  (--  _AAAAAAA__A_/                  |
  .    /./.+-  . .- /  +--  - .     \______________//_              \_______
  (__ ' /x  / x _/ (                                  \___'          \     /
 , x / ( '  . / .  /                                      |           \   /
    /  /  _/ /    +                                      /              \/
   '  (__/                                             /                  \
```